src/stories/alertMessage.stories.tsx

```diff
@@ -24,7 +24,17 @@ export default {
   },
 }
 
-const Template = (args) => {
+interface Props {
+  storyName: string
+  text: string
+  children: ReactNode
+  icon: () => ReactNode
+  fullWidth?: boolean
+  type: "error" | "warning" | "information" | "success"
+  alignCenter?: boolean
+}
```

Comment:
I'm guessing without fact checking here - most of these are probably defined in the component itself. can we rely on extending the existing type of the component to prevent duplication?

for example, if we'd introduce a new type of alert message, we'd currently need to add it in two places

Reply:
(this applies to many stories below)

Reply:
Yeah, that was my original idea, to extend the interface, but then I decided to just use the props that were actually used, not sure why. Thanks for the correction

Reply:
thanks, this looks much cleaner and easier to maintain now